lib/ddg-atb.js

```diff
@@ -16,7 +16,8 @@
 
 "use strict";
 
-const {Cu} = require("chrome");
+const {Cu, CC} = require("chrome");
+const XMLHttpRequest = CC("@mozilla.org/xmlextras/xmlhttprequest;1");
```

Comment:
@jdorweiler I do not want to slow the progress here down, but I believe it would be better to use a higher level API here (such as https://developer.mozilla.org/en-US/Add-ons/SDK/High-Level_APIs/request), rather than using this extra low level API.